### DISCUSSION TITLE
RTSP proxy UDP multicast configuration

### DISCUSSION THREAD

**Discussion by robopp55** [bluenviron/mediamtx#654] [Questions] [answered]
Hi,

First, thank you for creating this project. Very useful for our use case where we have an RTSP server on a bandwidth limited link and then are using your rtsp-simple-server in proxy mode to serve up the stream to multiple clients on a AP LAN. Our RTSP is real-time so no need to buffer on the client side, or start or stop the stream so an UDP multicast solution seems like it's the perfect setup for our configuration. I tried to set it up using the configuration.yml file, but couldn't figure it out. Can you provide some guidance on if this is possible and, if so, what the configuration should be?

TIA!

**Comment by aler9** [comment, marked as answer]
The readme has been improved with a Multicast section, please read it:

https://github.com/aler9/rtsp-simple-server#multicast

[a comment by an automated account is not shown]
